src/mzn_bench/analysis/report_status.py

```diff
@@ -26,7 +26,7 @@ def report_status(
 
             seen_status.add(row["status"])
             key = tuple(key)
-            time = float(row["time"])
+            time = float(row["time"] if row["time"] else 0)
```

Comment:
Could you make the if-statement more specific? I'd rather check `if row["time"] == ""` so it is more clear what the situation is.